designer/server/src/common/nunjucks/context/index.js

```diff
@@ -31,6 +31,7 @@ export async function context(request) {
   const credentials = request ? await getUserSession(request) : undefined
 
   return {
+    env,
     breadcrumbs: [],
     config: {
       phase,
```

Comment:
Mind sticking this in `config` so it's still `{{ config.env }}` in Nunjucks too?

```suggestion
    breadcrumbs: [],
    config: {
      env,
      phase,
```

Reply:
agree